display_information/display-annotation/src/main/java/com/esri/samples/display_annotation/DisplayAnnotationSample.java

```diff
@@ -54,19 +58,35 @@ public void start(Stage stage) {
             mapView.setMap(map);
 
             // create a feature layer from a feature service
-            FeatureLayer riverFeatureLayer = new FeatureLayer(new ServiceFeatureTable("https://services1.arcgis.com/6677msI40mnLuuLr/arcgis/rest/services/East_Lothian_Rivers/FeatureServer/0"));
+            riverFeatureLayer = new FeatureLayer(new ServiceFeatureTable("https://services1.arcgis.com/6677msI40mnLuuLr/arcgis/rest/services/East_Lothian_Rivers/FeatureServer/0"));
 
             // add the feature layer to the map
             map.getOperationalLayers().add(riverFeatureLayer);
 
             // create an annotation layer from a feature service
-            AnnotationLayer annotationLayer = new AnnotationLayer("https://sampleserver6.arcgisonline.com/arcgis/rest/services/RiversAnnotation/FeatureServer/0");
+            annotationLayer = new AnnotationLayer("https://sampleserver6.arcgisonline.com/arcgis/rest/services/RiversAnnotation/FeatureServer/0");
 
             // add the annotation layer to the map
             map.getOperationalLayers().add(annotationLayer);
 
+            // check that the feature layer has loaded
+            // this event gets triggered asynchronously when the feature layer that has been added to the map is loaded
```

Comment:
I'm thinking a slight rewording of the comments here and throughout the PR, what do you think?
```suggestion
            // check that the feature layer has loaded by adding a done loading listener (triggered asynchronously)
            // check for the load status of the layer and if it hasn't loaded, report an error
```

Reply:
I'm having some problems with this wording, because to me it reads as if the action of adding the listener is what's triggered asynchronously. Also the repetition of 'check' in the beginning bugs me a bit. How about:
```suggestion
            // add a done loading listener, with a runnable that gets triggered asynchronously when the feature layer has loaded
            // check for the load status of the layer and if it hasn't loaded, report an error
```
A bit wordy, but hopefully clearer?

Reply:
@JonLavi yeh I like this wording! Minor change I'd make would be to replace "gets" with "is", just reads a little better I find.